### ISSUE TITLE
Move directory structure to the gradle standard

### ISSUE THREAD

**Issue by pavelber** [spidru/JGribX#4]
It means create "java" folder under "main"
Otherwise gradle build creates empty jar

**Comment by pavelber** [comment]
Now gradle build is not working. Just create 'java' dir under src/main and move 'mt' dir (package) to it

**Comment by spidru** [comment]
Hi @pavelber,

Sorry for the delay, I've been trying out some stuff with gradle when I get the chance. It seems that moving everything into a "java" directory causes the following exception:

```
java.lang.SecurityException: Prohibited package name: java.mt.edu.um.cf2.jgribx
```

Have you encountered this issue?

**Comment by pavelber** [comment]
I'll try to make a pull request

**Comment by spidru** [comment]
Resolved by #9, so I'll close this issue.
